docs/client-sdk/javascript/tutorials/build-key-xmtp-chat-features-in-a-lens-app.md

```diff
@@ -224,7 +232,7 @@ To see how starting a conversation with a Lens profile is implemented in Lenster
 
 ## Stream new Lens DM conversations
 
-When building with XMTP to provide a chat experience in your Lens app, implement conversation streaming to ensure that your app can display new Lens conversations without requiring users to refresh your app first.
+When building with XMTP to provide a chat experience in your Lens app, implement conversation streaming to ensure that your app can detect when a new Lens conversation is starting without requiring users to refresh your app first.
```

Comment:
I made this update based on Nick's reply to a question in Discord: https://discord.com/channels/831836269558235136/993229186951028796/1067158197896499401